### PR TITLE
Project : Image Search Engine Frontend Project

### PR DESCRIPTION
**Pull request by GAVINESHWAR**
Closes: 

issue : #351 
- **Title** : Image Search Engine
- **Name:** : G AVINESHWAR
- **Idenitfy yourself:** : GSSOC

### Describe the add-ons or changes you've made 📃
I am contributed the Image Search Engine project to this WebMasterLog it is an Front end project developed using html, css, javascript and Api integration.

### Screenshots 📷

Screenshot 2024-06-21 115053
Screenshot 2024-06-21 115223

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] My changes generate no new warnings.

### Note to reviewers 📄

I PR the issue @Avdhesh-Varshney check my changes and merge the changes to make contribution to our project sir. 

Thank you

### CODE REVIEW

File: Front-end-Projects/Intermediate/Image-Search-Engine/README.md
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Intermediate/Image-Search-Engine/README.md:1]: follow readme template and naming convention of ss is `screenshot.webp` not `Screenshot.webp`.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@GAVINESHWAR keep your project according to the tags assigned on your issue. So, keep it inside `Vanilla-JS` and `Basic` folder.

File: Front-end-Projects/Intermediate/Image-Search-Engine/README.md
Comment (GAVINESHWAR) [reply Front-end-Projects/Intermediate/Image-Search-Engine/README.md:1]: @Avdhesh-Varshney , I done all the changes which were instructed , so please check my changes and merge the PR sir.

Thank you

**Comment by GAVINESHWAR** [comment]
@Avdhesh-Varshney I closed the PR because of conflicts  and added new PR for this issue , so please check the changes of PR : #392 and merge the changes sir.

Thank you

**Pull request closed without merging**